docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -783,6 +787,67 @@ class MyBean {
 [[websocket-next-security]]
 === Security
 
+Security capabilities are provided by the Quarkus Security extension.
+Any xref:security-identity-providers.adoc[Identity provider] can be used to convert authentication credentials on the initial HTTP request into a `SecurityIdentity` instance.
+The `SecurityIdentity` is then associated with the websocket connection.
+Authorization options are demonstrated in following sections.
+
+NOTE: When an OpenID Connect extension, `quarkus-oidc`, is used and token expires, Quarkus automatically closes connection.
+
+[[secure-http-upgrade]]
+==== Secure HTTP upgrade
+
+An HTTP upgrade is secured when a standard security annotation is placed on an endpoint class or an HTTP Security policy is defined.
+The advantage of securing HTTP upgrade is less processing, the authorization is performed early and only once.
+You should always prefer HTTP upgrade security unless, like in the example above, you need to perform an action on error or a security check based on the payload.
```

Comment:
> like in the example abov

Which example is it?

Reply:
now it is example in the next section, I'll refer to that section in gist, good catch